galaxy_ng/tests/unit/api/base.py

```diff
@@ -86,7 +86,7 @@ def _create_group(scope, name, users=None, perms=[]):
         return group
 
     @staticmethod
-    def _create_namespace(name, groups=None):
+    def _create_namespace(name, groups=[]):
```

Comment:
default parameter values are globally shared, even between calls. I think it looks safe in the use here, but its generally a good idea to avoid mutable default values. It would be really easy for a future change to mutate this and that mutation leak into all other calls to the method.

Maybe revert it and add `if groups is None: groups = []`?

Reply:
it can be

```py
def _create_namespace(name, groups=None):
    groups = groups or []
```